scripts/deploy_nightly.sh

```diff
@@ -31,7 +31,8 @@ fi
 
 # Try to update the lean-x.y.z branch on mathlib. This could fail if
```

Comment:
We still cannot get of this file, right?

Reply:
I think we should keep it at least until `leanproject` is official.